Octokit.GraphQL.Core.Generation.UnitTests/Octokit.GraphQL.Core.Generation.UnitTests.csproj

```
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Sorry, my mistake in my previous comment. For projects that were targeting `netcoreapp`, those should stay as-is. But for projects that were targeting `netstandard1.1`, those should be updated to `netstandard2.0`.

Only class libraries can target `netstandard`; unit tests and other apps need to target a `netcoreapp`.

```suggestion
    <TargetFramework>netcoreapp3.1</TargetFramework>
```